index.js

```diff
@@ -296,12 +296,23 @@ module.exports = {
 
     // Add nonce to <script> tag inserted by Ember CLI to assert that test file was loaded.
     if (type === 'test-body-footer') {
+      let emberCliDependency = new VersionChecker(this.project).for(
+        'ember-cli'
+      );
+
       existingContent.forEach((entry, index) => {
-        if (
-          /<script>\s*Ember.assert\(.*EmberENV.TESTS_FILE_LOADED\);\s*<\/script>/.test(
+        let result;
+        if (emberCliDependency.exists() && emberCliDependency.lt('3.25.1')) {
+          result = /<script>\s*Ember.assert\(.*EmberENV.TESTS_FILE_LOADED\);\s*<\/script>/.test(
+            entry
+          );
+        } else {
+          result = /<script>.*?Ember\.assert\(.*EmberENV\.TESTS_FILE_LOADED\);\s*}\);<\/script>/.test(
             entry
-          )
-        ) {
+          );
+        }
```

Comment:
Do I get it right that this is adding compatibility with Ember CLI >= 3.25.1 and therefore fixes floating dependencies test scenario?

Reply:
Just reviewed #197, which is about this change only. I guess you just included that one here to ease testing. I have just merged it.